app/services/setting_registry.rb

```diff
@@ -89,10 +89,9 @@ def set_user_value(name, value)
 
   # Returns all the categories used for settings
   def categories
-    sticked_general = { 'general' => nil }
-    sticked_general.merge!(Hash[@settings.map { |_name, definition| [definition.category_name, definition.category_label] }])
-    sticked_general.delete('general') if sticked_general['general'].nil?
```

Comment:
This is now removed. Are you assuming that `general` is always present?

Reply:
hmm, looks like this is still needed in case a search hides the general category. will fix. (tbh, the `new` in `subset` is quite confusing for a singleton...)